mne_bids/tests/test_write.py

```diff
@@ -498,20 +499,24 @@ def test_kit(_bids_validate):
         raw_fname, mrk=[hpi_pre_fname, hpi_post_fname], elp=electrode_fname,
         hsp=headshape_fname)
     write_raw_bids(raw,
-                   kit_bids_basename.replace('sub-01', 'sub-%s' % subject_id2),
+                   kit_bids_basename.replace('sub-01', f'sub-{subject_id2}'),
                    bids_root, events_data=events_fname, event_id=event_id,
                    overwrite=False)
 
     _bids_validate(bids_root)
     # ensure the marker files are renamed correctly
-    marker_fname = make_bids_basename(
-        subject=subject_id2, session=session_id, task=task, run=run,
-        suffix='markers.sqd', acquisition='pre',
-        prefix=os.path.join(bids_root, 'sub-02', 'ses-01', 'meg'))
+    marker_fname = marker_fname.copy()
+    marker_fname.acq = 'pre'
+    marker_fname.sub = subject_id2
+    marker_fname.prefix = op.join(bids_root, 'sub-02', 'ses-01', 'meg')
+    # marker_fname = make_bids_basename(
+    #     subject=subject_id2, session=session_id, task=task, run=run,
+    #     suffix='markers.sqd', acquisition='pre',
+    #     prefix=os.path.join(bids_root, 'sub-02', 'ses-01', 'meg'))
```

Comment:
what is this for???

Reply:
Sorry commented out, but forgot to remove.